src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Queries/LoadProductPricesRequest.cs

```diff
@@ -0,0 +1,11 @@
+using VirtoCommerce.CoreModule.Core.Common;
+using VirtoCommerce.ExperienceApiModule.Core.Infrastructure;
+
+namespace VirtoCommerce.XDigitalCatalog.Queries
+{
+    public class LoadProductPricesRequest : IQuery<LoadProductPricesResponce>
```

Comment:
All read operations must have the suffix  `Query` and implement `IQuery<TResponsetype>`